README.rst

```diff
@@ -59,104 +61,600 @@ Sample Output
+    'HOMEBREW_CELLAR'                     : '"/home/linuxbrew/.linuxbrew/Cellar"'
+    'HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS' : '3650'
+    'HOMEBREW_NO_AUTO_UPDATE'             : '1'
+    'HOMEBREW_PREFIX'                     : '"/home/linuxbrew/.linuxbrew"'
+    'HOMEBREW_REPOSITORY'                 : '"/home/linuxbrew/.linuxbrew/Homebrew"'
+    'INVOCATION_ID'                       : 'b7e3f0962f764bb4821eb641eff332da'
+    'ImageOS'                             : 'ubuntu20'
+    'ImageVersion'                        : '20210302.0'
+    'JAVA_HOME'                           : '/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64'
+    'JAVA_HOME_11_X64'                    : '/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64'
+    'JAVA_HOME_8_X64'                     : '/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64'
+    'JOURNAL_STREAM'                      : '9:19343'
+    'LANG'                                : 'C.UTF-8'
+    'LD_LIBRARY_PATH'                     : '/opt/hostedtoolcache/Python/3.9.2/x64/lib'
+    'LEIN_HOME'                           : '/usr/local/lib/lein'
+    'LEIN_JAR'                            : '/usr/local/lib/lein/self-installs/leiningen-2.9.5-standalone.jar'
+    'PATH'                                : '/opt/hostedtoolcache/Python/3.9.2/x64/bin:/opt/hostedtoolcache/Python/3.9.2/x64:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/opt/pipx_bin:/usr/share/rust/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin'
+    'PERFLOG_LOCATION_SETTING'            : 'RUNNER_PERFLOG'
+    'PIPX_BIN_DIR'                        : '/opt/pipx_bin'
+    'PIPX_HOME'                           : '/opt/pipx'
+    'POWERSHELL_DISTRIBUTION_CHANNEL'     : 'GitHub-Actions-ubuntu20'
+    'PWD'                                 : '/home/runner/work/python-info-action/python-info-action'
+    'RUNNER_OS'                           : 'Linux'
+    'RUNNER_PERFLOG'                      : '/home/runner/perflog'
+    'RUNNER_TEMP'                         : '/home/runner/work/_temp'
+    'RUNNER_TOOL_CACHE'                   : '/opt/hostedtoolcache'
+    'RUNNER_TRACKING_ID'                  : 'github_eb40fa08-b43b-4cc2-9f54-726413848d06'
+    'RUNNER_USER'                         : 'runner'
+    'RUNNER_WORKSPACE'                    : '/home/runner/work/python-info-action'
+    'SELENIUM_JAR_PATH'                   : '/usr/share/java/selenium-server-standalone.jar'
+    'SHLVL'                               : '1'
+    'SWIFT_PATH'                          : '/usr/share/swift/usr/bin'
+    'USER'                                : 'runner'
+    'VCPKG_INSTALLATION_ROOT'             : '/usr/local/share/vcpkg'
+    '_'                                   : '/opt/hostedtoolcache/Python/3.9.2/x64/bin/python'
+    'pythonLocation'                      : '/opt/hostedtoolcache/Python/3.9.2/x64'
```

Comment:
Maybe just start with 2 or 3 env vars and then cut the output ... same for workflow details

below is just an example.,. but i would cut a lot :)
```suggestion
   ...
```